workflow/rules/common.smk

```diff
@@ -231,6 +231,22 @@ def get_varlociraptor_preprocessing_input(wildcards, bai=False):
     )
 
 
+def get_primer_bed():
+    if config["primers"]["trimming"].get("primers_fa2", ""):
+        return "results/primers/primers.bedpe"
+    else:
+        return "results/primers/primers.bed"
+
+
+def get_target_regions():
+    if config["target_regions"].get("bed", ""):
+        if ["target_regions"].get("liftover_chain", ""):
```

Comment:
If possible, I'd specify the genome version of the `"target_regions"` file here and then automatically download the correct liftover file based on this genome versions and the genome version used for mapping in the workflow.

Reply:
Should be possible as long the Syntax would be like `<source>To<target>` as that seems to the the syntax which is used by [ucsc](https://hgdownload.soe.ucsc.edu/goldenPath/hg19/liftOver/)

Reply:
Ah right, they have their own genome version naming scheme. So this will not go together with our genome version specification. So either we map genome version specifications from one to the other, but would also have to do this for other species, or we do one of these:

1. The user does have to specify the liftover file, like you suggested.
2. We find another smart way of determining the regions of interest automatically. Maybe something that uses `mosdepth` to get the regions of the genome that are covered, and of those select only the regions that overlap with at least one of the primers?

Reply:
Yeah, it's a good start. But the danger is that it goes out of sync with the general genome specified for the workflow. Maybe something to discuss with Johannes at some point. But we can leave it like this, for now, and touch it again once we have fgbio working for single primers.